src/main/resources/db/changelog/changes/01-create-books-table.yaml

```diff
@@ -1,10 +1,10 @@
 databaseChangeLog:
   - changeSet:
-      id: create-book-table
+      id: create-books-table
       author: VolandevlodD
       changes:
         - createTable:
-            tableName: book
+            tableName: books
```

Comment:
Try to avoid editing previous migrations, it's better to create a new one for any future modifications. (In this PR you can leave it like this)

Reply:
ok, thx)